### PR TITLE
Switched from Unity's classic input system to the new Input System package

### PR DESCRIPTION
**Pull request by mattkwilson**
INTG-105

Related to: https://github.com/inworld-ai/inworld-unity-core/pull/60

### CODE REVIEW

File: Assets/Inworld/Inworld.Assets/Scripts/PlayerControl/InworldCameraController.cs [hunk 1]
```diff
@@ -35,25 +33,40 @@ public class InworldCameraController : MonoBehaviour
...
+            m_SpeedUpInputAction = InworldAI.InputActions["SpeedUp"];
+            m_SpeedInputAction = InworldAI.InputActions["Speed"];
+            m_MoveInputAction = InworldAI.InputActions["Move"];
+        }
+
         void Update()
         {
             // Hide and lock cursor when right mouse button pressed
-            if (playerInput.actions["LeftClick"].WasPressedThisFrame())
+            if (m_LeftClickInputAction.WasPressedThisFrame())
```
Comment (Juniverse) [inline Assets/Inworld/Inworld.Assets/Scripts/PlayerControl/InworldCameraController.cs:54]: Add null protection for those `m_XXAction` after fetched.

File: Assets/Inworld/Inworld.Samples.RPM/Scripts/UI/MultiCharCanvas.cs [hunk 2]
```diff
@@ -30,9 +29,10 @@ protected virtual void Awake()
             string bindingName = switchSelectionMethodActionPath.Substring(switchSelectionMethodActionPath.IndexOf('/') + 1);
             k_Instruction = $"Press <color=green>\"{bindingName[0].ToString().ToUpper() + bindingName.Substring(1)}\"</color> to switch character selection method.\n";
 
-            string bindingPath0 = m_CharacterSelectInputAction.action.bindings[0].path;
-            string bindingPath1 = m_CharacterSelectInputAction.action.bindings[1].path;
-            string bindingPath2 = m_CharacterSelectInputAction.action.bindings[2].path;
+            InputAction characterSelectionInputAction = InworldAI.InputActions["CharacterSelect"];
+            string bindingPath0 = characterSelectionInputAction.bindings[0].path;
+            string bindingPath1 = characterSelectionInputAction.bindings[1].path;
+            string bindingPath2 = characterSelectionInputAction.bindings[2].path;
```
Comment (Juniverse) [inline Assets/Inworld/Inworld.Samples.RPM/Scripts/UI/MultiCharCanvas.cs:35]: Why is there hardcoded 3 items? If that's only for the display of the `k_SelectByKey`, no need to update that.

**Review by Juniverse — APPROVED** [review]

File: Assets/Inworld/Inworld.Samples.RPM/Scripts/UI/MultiCharCanvas.cs (hunk 2 above)
Comment (mattkwilson) [reply Assets/Inworld/Inworld.Samples.RPM/Scripts/UI/MultiCharCanvas.cs:35]: I updated that so we can have the k_SelectByKey string be dynamic to whatever bindings are being used for that input action.

Comment (Juniverse) [reply Assets/Inworld/Inworld.Samples.RPM/Scripts/UI/MultiCharCanvas.cs:35]: Yeah no need to overdesign

Comment (Juniverse) [reply Assets/Inworld/Inworld.Samples.RPM/Scripts/UI/MultiCharCanvas.cs:35]: When we gave them the SDK, as long as the demo is working. Any change after that they will understand the UI inconsistancy themselves

**Pull request merged**